packages/just-scripts/src/monorepo/getStackDiffs.ts

```diff
@@ -0,0 +1,47 @@
+import fs from 'fs-extra';
+import glob from 'glob';
+import { downloadPackage, exec, paths, encodeArgs } from 'just-scripts-utils';
+import path from 'path';
+import { DiffInfo } from './DiffInfo';
+import { diff_match_patch, patch_obj, Diff } from 'diff-match-patch';
+
+export async function getStackDiffs(stack: string, fromVersion: string, toVersion: string): Promise<DiffInfo> {
+  const packagePaths = await Promise.all([downloadPackage(stack, fromVersion), downloadPackage(stack, toVersion)]);
```

Comment:
What if downloadPackage fails?

Reply:
What if!! yes, we need to handle this case!!

Reply:
We actually do handle this on the upgradeStackTask side.